<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RetrainDlg</class>
 <widget class="QDialog" name="RetrainDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Failed Train Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Image Failed at this tile
 when it was expected to pass.
     What is the Tile?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NormalButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>NORMAL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FlawButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>FLAW</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AbortButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>ABORT</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
